src/routes/task/task.routes.ts

```diff
@@ -0,0 +1,94 @@
+import {Router} from 'express'
+import Task from './taskModel'
+
+const router = Router()
+
+router.post('/api/tasks', async(req, res) => {
+	try {
+		const {text} = req.body
+		if( /^\s*$/.test(text)) {
+			return res.status(400).json({message: 'Text is empty!'})
+		}
+		const data = new Task({
+			text: req.body.text,
+			date: new Date().toLocaleString(),
+			description: '',
+			taskStatus: 'todo',
+			userId: req.user._id
+		})
+		const dataToSave = data.save()
+		res.status(200).json(dataToSave)
+	}
+	catch(error) {
+		res.status(400).json({message: error.message})
+	}
+})
+router.get('/api/tasks', async(req, res) => {
+	try {
+		const data = await Task.find({userId: req.user._id})
+		if(data) {
+			res.json(data)
+		} else {
+			res.status(404).send({message: 'Tasks do not exists!'})

```

Comment:
Jeżeli odpytujesz o wszystkie zasoby danego użytkownika to zwrócenie pustej tablicy, jeżeli nie ma żadnych jest ok

Reply:
dodane